src/lib/sql/sql_translator.cpp

```diff
@@ -1335,27 +1335,27 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_create_table(const hs
           column_definition.data_type = DataType::Double;
           break;
         case hsql::DataType::CHAR:
-          std::cout << "WARNING: Ignoring the length of CHAR as Hyrise has no way of working with it\n";
+          std::cout << "WARNING: Ignoring the length of CHAR. Hyrise's strings are not length-limited\n";
           column_definition.data_type = DataType::String;
           break;
         case hsql::DataType::VARCHAR:
-          std::cout << "WARNING: Ignoring the length of VARCHAR as Hyrise has no way of working with it\n";
+          std::cout << "WARNING: Ignoring the length of VARCHAR. Hyrise's strings are not length-limited\n";
           column_definition.data_type = DataType::String;
           break;
         case hsql::DataType::TEXT:
-          std::cout << "WARNING: Ignoring the length of TEXT as Hyrise has no way of working with it\n";
```

Comment:
Wait, as far as I remember, TEXT does not have an optional size. At least in the major DBMSs. So actually, TEXT is pretty much what we do in Hyrise as the default.